### ISSUE TITLE
0.2.40 introduced include issue

### ISSUE THREAD

**Issue by tytan652** [zapping-vbi/zvbi#40]
0.2.40 moved it's version macros to a generated file named `version.h`. This file is not installed making the use of `libzvbi.h` impossible.

Also zvbi seems to install its header directly in the include folder (no subfolder) meaning that the version header must not be named `version.h`.

**Comment by JuhaJGamer** [comment]
Yeah, I have to say this seems absolutely critical since it breaks the entire library and all of everything ever trying to compile against it.
